src/main/java/graphql/GraphQL.java

```diff
@@ -1,22 +1,7 @@
 package graphql;
 
-import graphql.execution.AbortExecutionException;
-import graphql.execution.AsyncExecutionStrategy;
-import graphql.execution.AsyncSerialExecutionStrategy;
-import graphql.execution.DataFetcherExceptionHandler;
-import graphql.execution.Execution;
-import graphql.execution.ExecutionId;
-import graphql.execution.ExecutionIdProvider;
-import graphql.execution.ExecutionStrategy;
-import graphql.execution.SimpleDataFetcherExceptionHandler;
-import graphql.execution.SubscriptionExecutionStrategy;
-import graphql.execution.ValueUnboxer;
-import graphql.execution.instrumentation.ChainedInstrumentation;
-import graphql.execution.instrumentation.DocumentAndVariables;
-import graphql.execution.instrumentation.Instrumentation;
-import graphql.execution.instrumentation.InstrumentationContext;
-import graphql.execution.instrumentation.InstrumentationState;
-import graphql.execution.instrumentation.SimpleInstrumentation;
+import graphql.execution.*;
+import graphql.execution.instrumentation.*;
```

Comment:
Sorry we require explicit imports - if you are using IDEA set them to 999 before using * for this project